### DISCUSSION TITLE
Update sortable documentation with better examples for Handle and Remove

### DISCUSSION THREAD

**Discussion by zampa00** [clauderic/dnd-kit#563] [Q&A]
When checking the [examples documentation](https://5fc05e08a4a65d0021ae0bf2-bphcuqqqih.chromatic.com/?path=/docs/presets-sortable-vertical--removable-items), it's shown that it's possible to create an item with a draggable handle or a button to remove the item.

When clicking on "Show code" for Drag Handle, it only shows `() => <Sortable {...props} handle />`, but it looks like Sortable is not an importable component, but just a component created to test the implementation and produce a playable documentation.

If there is an easy way to implement a drag handle, it should be pointed out somewhere in the documentation.

**Comment by mirorauhala** [comment]
There's some example code given on the draggable documentation page. Read more at https://docs.dndkit.com/api-documentation/draggable#flexibility
